src/main/java/demohometask/demohometask/Match.java

```diff
@@ -0,0 +1,92 @@
+package demohometask.demohometask;
+
+import org.springframework.stereotype.Component;
+
+import java.io.FileOutputStream;
+import java.io.ObjectOutputStream;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Random;
+
+@Component
+public class Match {
+    private static int round;
+    private static int numberOfRounds = 3;
+    private static String score;
+
+
+    public static List<Team> createListOfTeams(){
+        String[] names = {"John", "Kevin", "Bruno", "Max", "Mike", "Jack", "Baylor", "Villanova"};
+        String[] teams = {"Team_name_1", "Team_name_2", "Team_name_3", "Team_name_4", "Team_name_5", "Team_name_6", "Team_name_7", "Team_name_8"};
+        List<Team> listOfTeams = new LinkedList();
+
+        for(int i =0; i<names.length; i++){
+            listOfTeams.add(new Team(teams[i], names[i], names[i]));
+        }
+        return listOfTeams;
+    }
+
+    public static void start(){
```

Comment:
Don't use static methods in Spring component. If you need to invoke some method from component just inject it into relevant place and use